src/service/core/config/helpers.py

```
@@ -267,7 +267,7 @@ def _update_backend_helper(
     params.append(configs['name'])
 
     update_cmd = (
-        f'UPDATE backends SET {", ".join(values)} WHERE name = %s RETURNING name;'
+        f'UPDATE backends SET {', '.join(values)} WHERE name = %s RETURNING name;'
```

Ruff (0.15.10)

[error] Possible SQL injection vector through string-based query construction

(S608)